packages/base/src/Device.js

```diff
@@ -805,6 +805,97 @@ const isMobile = () => {
 	return Device.browser.mobile;
 };
 
+//* ******* Match Media ********
```

Comment:
I don't think this code belongs to `Device.js`. It may be so in OpenUI5, but this leads to enormous classes/modules that do a lot of unrelated or loosely related things. Additionally, the term "media" is confusing for this kind of functionality.

Please move to a new module, f.e. called: `MediaRange.js`, detached from the `Device` namespace. It should export 2 functions: `initRangeSet` and `getCurrentRange`